x-pack/solutions/observability/plugins/streams_app/public/components/schema_editor/flyout/ecs_recommendation.tsx

```diff
@@ -34,7 +34,13 @@ export const EcsRecommendation = ({
   return (
     <EuiText size="xs">
       {`${EcsRecommendationText}: `}
-      {isLoading ? LoadingText : recommendation !== undefined ? recommendation : UknownEcsFieldText}
+      <EuiBadge color="success">
+        {isLoading
+          ? LoadingText
+          : recommendation !== undefined
+          ? recommendation
+          : UknownEcsFieldText}
+      </EuiBadge>
```

Comment:
Showing a success badge when there might be an "Unknown" text doesn't seem very successful 😅